tests/test_isamples_metadata.py

```diff
@@ -202,19 +202,19 @@ def test_geome_transformer_for_identifier():
 
 OPENCONTEXT_test_values = [
     (
-        "./test_data/OpenContext/raw/ark-28722-k2b570022.json",
-        "./test_data/OpenContext/test/ark-28722-k2b570022-test.json",
-        "2021-06-27T19:54:46Z",
+        "./test_data/OpenContext/raw/ark-28722-k2b85cg1p.json",
```

Comment:
Add new test cases due to the change in OpenContext mapping. The previous test cases has item category values that invoked the machine learning model, which is not possible to test on.